### PR TITLE
Build metadata rework

### PR DESCRIPTION
**Pull request by arewm**
Picking up from an accidental merge of https://github.com/konflux-ci/docs/pull/80

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc [hunk 1]
```diff
@@ -130,13 +170,16 @@ Example output:
...
-   └── 🍒 sha256::<output omitted>
 └── 🔐 Signatures for an image tag: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg:sha256-:<output omitted>.sig
  └── 🍒 sha256::<output omitted>
 └── 📦 SBOMs for an image tag: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg:sha256-:<output omitted>.sbom
   └── 🍒 sha256:<output omitted>
 --
 
+== Inspecting the downloaded provenance
+
+The generated provenance includes information from the Tekton PipelineRun that generated the attested artifact. It
```
Comment (Chr1st1anSears) [inline docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc:181]: This sentence is a little hard to follow. I actually think if you just delete the first "generated," it would help. Also I think the "It" at the end is a typo.

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc [hunk 2]
```diff
@@ -1 +1,56 @@
-= Inspecting provenance and attestations
\ No newline at end of file
+= Inspecting provenance and attestations
+
+We are committed to providing exceptional security with {ProductName} by hardening the build platform and providing transparency into the build process and artifact's composition. By hardening the build platform, we enable multiple builds to run on a shared platform while preventing tampering from other running builds (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the build process]). By providing transparency, we enable xref:/advanced-how-tos/managing-compliance-with-ec.adoc[policy evaluation] and auditability on the actual build process (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the package selection]). This commitment means that {ProductName} conforms to the link:https://slsa.dev/spec/v1.0/levels#build-l3[SLSA Build Level 3] requirements.
```
Comment (Chr1st1anSears) [inline docs/modules/ROOT/pages/how-tos/metadata/index.adoc:3]: The opening here is hard to follow. Mainly, the sentences are too long. Consider this alternative: 
"We are committed to providing exceptional security with {ProductName}. We harden the build platform, we provide transparency into the build process, and we expose composition of artifacts. 

To harden the build platform, we enable multiple builds to run on a shared platform while preventing those builds from tampering with each other. This defends against threats to the build process. To provide transparency, we enable you to evaluate policies and conduct audits on the build process. This defends against threats to package selection. 

Our commitment to security means that {ProductName} conforms to the SLSA Build Level 3 requirements.

Don't take our word for it --you can download and inspect these artifacts yourself!"

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc [hunk 3]
```diff
@@ -1 +1,56 @@
...
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:/getting-started/cli.adoc[Login] to {ProductName}.
+
+.Procedure
+
+In the CLI, complete the following steps to discover the associated metadata a component:
```
Comment (Chr1st1anSears) [inline docs/modules/ROOT/pages/how-tos/metadata/index.adoc:19]: "the associated metadata *for* a component"

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc [hunk 4]
```diff
@@ -1 +1,56 @@
...
+📦 Supply Chain Security Related artifacts for an image: quay.io/redhat-user-workload/workspace-tenant/application/component@sha256:<sha-digest>
+└── 📦 SBOMs for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.sbom
+   └── 🍒 sha256:<sbom-sha-digest>
+└── 💾 Attestations for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.att
+   └── 🍒 sha256:<attestation-sha-digest>
+└── 🔐 Signatures for an image tag: quay.io/redhat-user-workload/workspace-tenant/application/component:sha256-<sha-digest>.sig
+   └── 🍒 sha256:<signature-sha-digest>
+--
+
+The `cosign tree` command will show all metadata associated to the artifact via the referrers API. The SBOM artifact contains the Software Bill of MaterialsWe have provided further procedures for you to view and inspect the xref:/how-tos/metadata/attestations.adoc[attestations] and xref:/how-tos/metadata/sboms.adoc[SBOMs] by {ProductName}.
```
Comment (Chr1st1anSears) [inline docs/modules/ROOT/pages/how-tos/metadata/index.adoc:52]: "...Software Bill of Materials. We have provided..."

Currently missing a period and space there.

**Review by Chr1st1anSears — CHANGES_REQUESTED** [review]
Hey Andrew,

Thanks for requesting my review. I think you added a lot of helpful info and technical rigor to the docs. 

I left a collection of comments that basically all amount to the same  piece of feedback: try to simplify. Obviously this is hard when we are writing about complicated software. But shorter sentences and grammatically simpler phrasing always help. RH/IBM direct us to write in this "minimalist" style to make things easier on readers, especially those who are reading English as a second language.

I hope this review helps. Lmk if I can clarify anything.

File: docs/modules/ROOT/pages/how-tos/creating.adoc [hunk 5]
```diff
@@ -111,3 +111,73 @@ metadata:
 spec:
   url: "https://github.com/konflux-ci/anothertestrepo"
 --
+
+== Finding the built images
+
+After a pipeline completes with a built artifact, you may want to test the resulting image to ensure that it works properly. The `IMAGE_URL` Tekton result will be set to the pullspec for the image. This can be discovered using the UI or the CLI.
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/creating.adoc:117]: This is the `IMAGE_URL`. I didn't want to repeat the same beginning of the sentence twice. I can combine the sentences though.

[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/creating.adoc:117]: How about 
```suggestion
After a pipeline completes with a built artifact, you may want to test the resulting image to ensure that it works properly. The `IMAGE_URL` Tekton result (discoverable from the UI or CLI) should be set to the pullspec for the image.
```

File: docs/modules/ROOT/pages/how-tos/creating.adoc [hunk 6]
```diff
@@ -111,3 +111,73 @@ metadata:
...
+
++
+NOTE: Images build from pull request pipelines will be automatically deleted 5 days after they are built. This parameter can be changed by xref:/how-tos/configuring/customizing-the-build.adoc[customizing the build].
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/creating.adoc:120]:
```suggestion
NOTE: {ProductName} automatically deletes images built for PR pipelines five days after building them.
```
Sounds good to me!

File: docs/modules/ROOT/pages/how-tos/creating.adoc [hunk 7]
```diff
@@ -111,3 +111,73 @@ metadata:
...
+
+.Procedure
+
+In the console, complete the following steps to find the image pullspec for a completed PipelineRun:
+
+. Navigate to the *Activity* > *Pipeline runs* tab.
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. Find the *Results* section at the bottom of the page and look for the `IMAGE_URL` row. It should resemble `quay.io/redhat-user-workloads/workspace-tenant/application/component:tag`. The `IMAGE_DIGEST` provided can be used as an alternative mechanism for referencing the image. You will
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/creating.adoc:134]:
```suggestion
. Find the *Results* section at the bottom of the page and look for the `IMAGE_URL` row. It should resemble `quay.io/redhat-user-workloads/workspace-tenant/application/component:tag`. You can use the `IMAGE_DIGEST` provided as an alternate mechanism for referencing the image.
```
Thanks for catching the dangle. My mind was in circles trying to find the best restructure for the documentaiton.

File: docs/modules/ROOT/pages/how-tos/creating.adoc [hunk 8]
```diff
@@ -111,3 +111,73 @@ metadata:
...
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. Find the *Results* section at the bottom of the page and look for the `IMAGE_URL` row. It should resemble `quay.io/redhat-user-workloads/workspace-tenant/application/component:tag`. The `IMAGE_DIGEST` provided can be used as an alternative mechanism for referencing the image. You will 
+
+=== With the CLI
+
+After the build PipelineRuns are completed from git push events, the Components are updated with the location of the artifact in the OCI registry.
+
+The CLI
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/creating.adoc:140]: It certainly does.
```suggestion
```

File: docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc [hunk 9]
```diff
@@ -1,8 +1,53 @@
...
 
-We have also provided the following procedure for you to actually access the SLSA provenace that {ProductName} generates for each of your xref:../glossary/index.adoc#component[components]. Follow the instructions below to download your SLSA provenance.
+After you have found the image for a xref:/how-tos/creating.adoc#finding-the-built-image[recently completed build pipeline] and exported the value to the `IMAGE` environment variable, you are ready to start exploring the information stored in the attestations.
+
+== Inspect the SLSA provenance
+
+Tekton Chains produces a very detailed SLSA provenance. If we were to look at the output from the following provenance, we would find that it is over 2800 lines long!
```
[the remark this answers is not in the document]
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc:9]:
```suggestion
Tekton Chains produces a very detailed SLSA provenance. The output from the following provenance is over 2800 lines long!
```

File: docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc (hunk 1 above)
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc:181]: I think that this entire line was mistakenly included as well as the procedure above.

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc (hunk 2 above)
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/index.adoc:3]: Yeah, I was having difficulty with this paragraph. I was trying to do some parallel construction but that made it even harder. Are you recommending to remove the links as well? Do they add unnecessary complexity?

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc (hunk 3 above)
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/index.adoc:19]:
```suggestion
In the CLI, complete the following steps to discover the associated metadata for a component:
```

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc (hunk 4 above)
Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/index.adoc:52]:
```suggestion
The `cosign tree` command will show all metadata associated to the artifact via the referrers API. The SBOM artifact contains the Software Bill of Materials. We have provided further procedures for you to view and inspect the xref:/how-tos/metadata/attestations.adoc[attestations] and xref:/how-tos/metadata/sboms.adoc[SBOMs] by {ProductName}.
```

File: docs/modules/ROOT/pages/how-tos/metadata/index.adoc (hunk 2 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/how-tos/metadata/index.adoc:3]: Ahh nah I think the links are good! I was just being lazy and didn't include them in my alternative.

Also fwiw I think it's particularly difficult to write the opening for this doc. Yours gave a solid starting place/better technical info than what I had there before. I think this draft ^^ just breaks it down makes it more digestible.

Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/metadata/index.adoc:3]: Something like this?
```suggestion
We are committed to providing exceptional security with {ProductName}. We harden the build platform, we provide transparency into the build process, and we expose the composition of artifacts.

To harden the build platform, we enable multiple builds to run on a shared platform while preventing those builds from tampering with each other. This defends against link:https://slsa.dev/spec/v1.1/threats-overview[threats to the build process]. To provide transparency, we enable you to xref:/advanced-how-tos/managing-compliance-with-ec.adoc[evaluate policies] and conduct audits on the build process. This defends against link:https://slsa.dev/spec/v1.1/threats-overview[threats to the package selection].

Our commitment to security means that {ProductName} conforms to the link:https://slsa.dev/spec/v1.0/levels#build-l3[SLSA v1.0 Build Level 3] requirements.

Don't take our word for it --you can download and inspect these artifacts yourself!
```

**Review by Chr1st1anSears — APPROVED** [review]
